datasets/ar_res_reviews/README.md

```diff
@@ -0,0 +1,150 @@
+---
+annotations_creators:
+- unknown
+language_creators:
+- unknown
+languages:
+- ar
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 1k<n<10k
+source_datasets:
+- original
+task_categories:
+- text_classification
+task_ids:
+- sentiment-classification
+---
+
+# Dataset Card for MetRec
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Discussion of Social Impact and Biases](#discussion-of-social-impact-and-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Large Arabic Sentiment Analysis Resources](https://github.com/hadyelsahar/large-arabic-sentiment-analysis-resouces)
+- **Repository:** [Large Arabic Sentiment Analysis Resources](https://github.com/hadyelsahar/large-arabic-sentiment-analysis-resouces)
+- **Paper:** [ Building Large Arabic Multi-domain Resources for Sentiment Analysis](https://github.com/hadyelsahar/large-arabic-sentiment-analysis-resouces/blob/master/Paper%20-%20Building%20Large%20Arabic%20Multi-domain%20Resources%20for%20Sentiment%20Analysis.pdf)
+- **Point of Contact:** [hady elsahar](hadyelsahar@gmail.com)
+
+### Dataset Summary
+
+Dataset of 8364 restaurant reviews from qaym.com in Arabic for sentiment analysis
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset is based on Arabic.
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point comprises of the following:
+
+- "polarity": which is a string value of either -1 or 1 indicating the sentiment around the review    
+
+- "text": is the review plain text of a restaurant in Arabic
+
+- "restaurant_id": the restaurant ID on the website
+
+- "user_id": the user ID on the website
```

Comment:
This section expects one example instance, you could print one example and paste that here

Reply:
done!

Reply:
oh wait I should run make and flake8 again?